File: readability/readability.c
#include <stdio.h>
#include <cs50.h>
#include <string.h>
#include <math.h>

int count_words(string text);

int main(void)
{
    //get a string from the user
    string text;
    text = get_string("Text: \n");

    printf("%s\n", text);
}

int count_letters(string text)
{
    for (int i = 0; i < strlen(text); i++)
    {
        count_words(text);
    }
}

int count_words(string text)
{
    int letters;
    if((text[i] >= 'a' && text[i] <= 'z') || (text[i] >= 'A' && text[i] <= 'Z'))
    letters++;
    printf("%i letters \n", letters);
}

File: sentimental-readability/readability.py
from cs50 import get_string
import string

# get input from user
text = get_string("Text: ")

# declare variables: words, letters and sentences so we can make the calculus
# to count words we will use spaces, so we begin with 1

letters = 0
words = 1
sentences = 0

for letter in text:
    if letter == "!" or letter == "?" or letter == ".":
        sentences += 1
    # the following avoid that other punctuation such as commas count as a letter:
    elif letter in string.punctuation:
        continue
    # 
    elif letter in string.whitespace:
        words += 1
